src/components/ui/table.tsx

```diff
@@ -82,7 +82,7 @@ React.TdHTMLAttributes<HTMLTableCellElement>
 >(({ className, ...props }, ref) => (
   <td
     ref={ref}
-    className={cn('p-2 align-middle [&:has([role=checkbox])]:pr-0', className)}
+    className={cn('border-b p-2 align-middle [&:has([role=checkbox])]:pr-0', className)}
```

Comment:
I would suggest making the group header slightly thinner to further differentiate it against normal rows.
`px-2 py-1` instead of `p-2`

Reply:
Yes, good suggestion. I'll leave p-2 here, as it will apply to all td's. And add px-2 py-1 in the chrome instead so it only will be applied to the group header.